Sources/SwiftyMocky/MockyAssert.swift

```diff
@@ -45,12 +51,33 @@ public func MockyAssert(
 private func XCTMockyAssert(
     _ expression: @autoclosure () -> Bool,
     _ message: @autoclosure () -> String = "Verify failed",
+    fileId: StaticString = #fileID,
+    filePath: StaticString = #filePath,
     file: StaticString = #file,
-    line: UInt = #line
+    line: UInt = #line,
+    column: UInt = #column
 ) {
-    #if canImport(XCTest)
-    XCTAssert(expression(), message(), file: file, line: line)
-    #else
+#if canImport(Testing)
+    if Test.current != nil {
+        if !expression() {
+            Issue.record(
+                Comment(rawValue: message()),
+                sourceLocation: .init(
+                    fileID: fileId.description,
+                    filePath: filePath.description,
+                    line: Int(line),
+                    column: Int(column)
+                )
+            )
+        }
+        return
+    }
+#endif
```

Comment:
The part above is the only thing that's needed to make the test correctly fail in Swift Testing. All the rest is just to make it show in the editor.